src/serlo-editor/plugins/text/hooks/use-slate-render-handlers.tsx

```diff
@@ -80,6 +80,31 @@ export const useSlateRenderHandlers = ({
           </MathElement>
         )
       }
+      if (element.type === 'gap') {
+        // TODO: Render <GapRenderer> here instead
+        // <GapRenderer> needs to ...
+        // - always show input fields
+        // - accept callback onChange that is passed to input element to update correctAnswer
+        return (
+          <span {...attributes} contentEditable={false}>
+            <input
+              value={element.correctAnswer}
+              size={element.correctAnswer.length}
+              className="rounded-full border border-brand bg-brand-50 px-2"
+              onChange={(evt) => {
```

Comment:
Super minor, but we already have `e` and `event` throughout the codebase, so maybe just use one of those instead of `evt`, for the sake of consistency.

Reply:
Changed it to `e`